### PR TITLE
Dev/cuda docker file

### PR DESCRIPTION
**Pull request by as6520**

### CODE REVIEW

**Comment by dstoup** [comment]
I think there is some work done here that might conflict with a change @Cookt2 made and merged recently though the PR doesn't show any. Otherwise, it looks good to me. I will wait for Taylor to chime in.

File: dockerfile [hunk 1]
```diff
@@ -1,20 +1,46 @@
 # Install KWIVER to /opt/kitware/kwiver
 # Use latest Fletch as base image (Ubuntu 18.04)
+ARG BASE_IMAGE=kitware/fletch:latest
```
Comment (Cookt2) [inline dockerfile]: This should be changed to: "kitware/fletch:latest-ubuntu18.04-py3"

With the latest changes to the fletch dockerfile, we are now producing different labeling syntax.

The default base fletch image is now "kitware/fletch:latest-ubuntu18.04-py3", I plan on removing "kitware/fletch:latest" altogether.

File: dockerfile [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+ARG PY_MAJOR_VERSION=3
+ARG ENABLE_CUDA=OFF
+ARG KWIVER_INSTALL=/opt/kitware/kwiver
 
+# Conditonal installtion of six
+RUN if [ "$PY_MAJOR_VERSION" = "2" ]; then \
+      pip install six; \
+    else \
+      pip3 install six; \
+    fi
```
Comment (Cookt2) [inline dockerfile]: Why are we installing six? I haven't had it in the previous dockerfiles.

**Comment by linus-sherrill** [comment]
Is this ready to go or is there still a problem?

**Comment by as6520** [comment]
The original intent of this PR was to provide a dockerfile that would build base images for DIVA. Thus the dockerfile using the components that diva requires to build. I discussed this PR with @Cookt2 and that is not the intent of the dockerfile. Thus I am closing the PR.

**Pull request closed without merging**